### ISSUE TITLE
[Bug] - Unresponsive piano notes

### ISSUE THREAD

**Issue by TheRyanReSeT** [Backdash/MonikaModDev#2660]
I noticed I wasn't able to play a few chords in the upper register of the piano.

Noting: CMaj7 and C7.

For you non-music people, in the default key configuration, it's 
CMaj7: "T" "U" "O" "["
C7: "T" "U" "O" "-"

The piano will only recognize just the C Major chord keys "T" "U" "O"
and will not play back the "[" and "-"
These two were the only ones I noticed not playable in combination with C Major.

**Comment by ThePotatoGuy** [comment]
Does your keyboard support N-key rollover?

**Comment by TheRyanReSeT** [comment]
I don't think so? I ran this test and at most I can do up to 6-keys
http://gadzikowski.com/nkeyrollover.html

I did notice that the keys weren't registering "T" "U" "O" "[" at the same time during the test like it was in-game.
BUT, the combination of "Y" "I" "P" "]" does work, registering 4 key presses. I don't know if it's something with my keyboard or some particular about these bottom 4 combinations:
CMaj7: "T" "U" "O" "["
C7: "T" "U" "O" "-"
